src/vcpkg-test/platform-expression.cpp

```diff
@@ -479,66 +515,113 @@ TEST_CASE ("invalid logic expression, use '|' instead of 'or'", "[platform-expre
 
 TEST_CASE ("unexpected character or identifier in logic expression", "[platform-expression]")
 {
-    auto m_expr = parse_expr("windows aND arm");
-    CHECK_FALSE(m_expr);
-    m_expr = parse_expr("windows a&d arm");
-    CHECK_FALSE(m_expr);
-    m_expr = parse_expr("windows oR arm");
-    CHECK_FALSE(m_expr);
-    m_expr = parse_expr("windows o|r arm");
-    CHECK_FALSE(m_expr);
+    {
+        auto m_expr = parse_expr("windows aND arm");
+        CHECK_FALSE(m_expr);
+    }
```

Comment:
```suggestion
    CHECK_FALSE(parse_expr("windows aND arm"));
```
I feel like spreading this out over 5 lines just adds boilerplate and does not improve legibility of the test.